lib/index.d.ts

```
@@ -0,0 +1,692 @@
+export interface getVoucherRequest {
+    Blockchain: string;
+    Code: string;
+    Version: string;
+}
+export interface getDomainRequest {
+    Blockchain: string;
+    Domain: string;
+    Version: string;
+}
+export interface getBlockchainsRequest {
+}
```

Biome (2.1.2)

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)